### PR TITLE
Upated Japanese translation

### PR DESCRIPTION
**Pull request by tkusano**
Deleted obsoleted messages which are mostly related to old onboarding messages.

### CODE REVIEW

**Comment by tkusano** [comment]
- Translate new messages
-  Replace 3 dots with HORIZONTAL ELLIPSIS

**Comment by tkusano** [comment]
@Hima-Zinn @dolciss @oboenikui @noritada @middlingphys @hibiki @reindex-ot @haoyayoi @vyv03354

Please review

**Comment by dolciss** [comment]
Thanks for the latest translation.
LGTM👍

**Review by Hima-Zinn — COMMENTED** [review]
Thank you for your work.

**Comment by tkusano** [comment]
Add new translations mainly related to https://github.com/bluesky-social/social-app/commit/3ece21cb45e8b74795f4eac33a1551b303196946 . Translated as follows.

deactivation -> 無効化
reactivation -> 再有効化

Changed 'マイアカウント' to 'アカウント' to simplify UI.

Also, changed half-width '()' to full-width '（）'.

File: src/locale/locales/ja/messages.po [hunk 1]
```diff
@@ -5089,9 +5043,9 @@ msgstr "サポートフォームは移動しました。サポートが必要な
 msgid "The Terms of Service have been moved to"
 msgstr "サービス規約は移動しました"
 
-#: src/screens/Onboarding/StepAlgoFeeds/index.tsx:141
-msgid "There are many feeds to try:"
-msgstr "試せるフィードはたくさんあります："
+#: src/screens/Settings/components/DeactivateAccountDialog.tsx:35
+msgid "There is no time limit for account deactivation, come back any time."
+msgstr "アカウントの無効化に期限はありません。いつでも戻ってこれます。"
```
Comment (dolciss) [inline src/locale/locales/ja/messages.po:5048]:
```suggestion
msgstr "アカウントの無効化に期限はありません。いつでも戻ってこられます。"
```

**Review by dolciss — COMMENTED** [review]
Thanks for the additional translation.[^1]
I would appreciate it if you could correct one minor point.

[^1]: 別で提案した「無効化」案を採用いただきありがとうございます。その際にあった復活できないinvalidate的な意味に取られる可能性も、UIに復活できる旨の説明があるため大丈夫かと私も考えています。

**Review by pfrazee — APPROVED** [review]
Thanks!!

File: src/locale/locales/ja/messages.po (hunk 1 above)
Comment (tkusano) [reply src/locale/locales/ja/messages.po:5048]: fixed in https://github.com/bluesky-social/social-app/pull/4477/commits/fa74fb9dfef6bdeeaf24c589ac52b0f221c51319 . thanks.

**Pull request merged**